tests/connectors/kubernetes_test.py

```diff
@@ -2013,11 +2015,11 @@ def namespace(self, kube: kubetest.client.TestClient) -> str:
     @pytest.mark.applymanifests("../manifests/sidecar_injection",
                                 files=["fiber-http_single_port.yaml"])
     @pytest.mark.parametrize(
-        "service, port",
+        "port, service",
         [
-            ('fiber-http', None),
-            ('fiber-http', 80),
-            ('fiber-http', 'http'),
+            (None, 'fiber-http'),
+            (80, 'fiber-http'),
+            ('http', 'fiber-http'),
```

Comment:
Why'd you flip this? It seems more intuitive to me as service then port?

Reply:
This resolves some of the flakiness in integration tests due to kubetest namespace collisions. What happens is that the parameter is appended to the test name then truncated to 43 characters and prepended with kubetest and appended with a timestamp. The result namespace name is `kubetest-test-inject-single-port-deployment-fiber-ht-` which is the same for all 3 parameterizations which causes conflicts when each of the test goes to create the namespace. You can see an example of this failure in this test run: https://github.com/opsani/servox/runs/3204520239

By reordering the parameters, each test parameterization produces a unique namespace name which prevents issues from tests trying to use the same namespace and/or tearing down the namespace set up by another test